### PR TITLE
Tweets develop

### PR DESCRIPTION
**Pull request by ali-moradzade**
Adding like/dislike ability, changing error messages to persian

### CODE REVIEW

File: src/controllers/tweets.controller.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            });
+    },
+
+    createTweet(req, res) {
+        const tweetProps = req.body;
+
+        tweetsService.createTweetService(tweetProps)
+            .then(tweet => {
+                if (tweet) {
+                    res.send(tweet);
```
Comment (arefehkompani) [inline src/controllers/tweets.controller.js]: We need `message` here.

File: src/controllers/tweets.controller.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            });
+    },
+
+    deleteTweet(req, res) {
+        const id = req.params.id;
+
+        tweetsService.deleteTweetService(id)
+            .then((tweet) => {
+                if (tweet) {
+                    res.send(tweet);
```
Comment (arefehkompani) [inline src/controllers/tweets.controller.js]: we need `message` here.

File: src/controllers/tweets.controller.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                console.log(err);
+                res.status(422).json({
+                    message: someThingWentWrongMsg
+                });
+            });
+    },
+
+    deleteTweet(req, res) {
+        const id = req.params.id;
+
```
Comment (arefehkompani) [inline src/controllers/tweets.controller.js:75]: I think we should check that the creator of the tweet is the same user who logged in.

**Review by arefehkompani — COMMENTED** [review]
Please add uploading images of the tweet.

File: src/controllers/tweets.controller.js (hunk 2 above)
Comment (ali-moradzade) [reply src/controllers/tweets.controller.js]: Done.

File: src/controllers/tweets.controller.js (hunk 3 above)
Comment (ali-moradzade) [reply src/controllers/tweets.controller.js:75]: Done

File: src/controllers/tweets.controller.js (hunk 1 above)
Comment (ali-moradzade) [reply src/controllers/tweets.controller.js]: Done

File: src/controllers/tweets.controller.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+            })
+            .catch(err => {
+                console.log(err);
+                res.status(400).json({
+                    message: invalidIdMsg
+                });
+            });
+    },
+
+    createTweet(req, res) {
```
Comment (arefehkompani) [inline src/controllers/tweets.controller.js:46]: Please get userId from authentication middleware not the body.

**Review by arefehkompani — APPROVED** [review]

**Pull request merged**